web-components/src/components/header/components/UserMenuItem.tsx

```diff
@@ -29,12 +30,8 @@ const UserMenuItem: React.FC<React.PropsWithChildren<UserMenuItemProps>> = ({
   disconnect,
   pathname,
   linkComponent,
+  userMenuItems,
```

Comment:
the component itself should have the name in the plural, right?
`<UserMenuItems />`